defaults/main.yml

```diff
@@ -86,7 +86,6 @@ jenkins2_plugins_suggested:
   - sonar
   - checkstyle
   - gitlab-plugin
-  - job-dsl
```

Comment:
this will break functionality added in pr #14

Reply:
this is a duplicate